plugins/jenkins-backend/src/service/router.ts

```diff
@@ -17,21 +17,25 @@
 import { errorHandler } from '@backstage/backend-common';
 import express from 'express';
 import Router from 'express-promise-router';
-import { Logger } from 'winston';
-import { JenkinsInfoProvider } from './jenkinsInfoProvider';
+import type { Logger } from 'winston';
+import type { JenkinsInfoProvider } from './jenkinsInfoProvider';
```

Comment:
Should be no need to explicitly specify `type` imports, it's detected during transpilation based on usage within the file anyway